java/geodesic-operations/README.md

```diff
@@ -0,0 +1,33 @@
+# Geodesic Operations 
+Demonstrates how to use the GeometryEngine to calculate a geodesic path between two points and measure its distance.
```

Comment:
I only just started doing this myself yesterday, but lets drop the "Demonstrates how to..." so it would just become. "Use the GeometryEngine to calculate..."